SpeechService/Translations.cs

```diff
@@ -728,32 +729,57 @@ public static string Humanize(decimal? value)
             }
 
             // Describe decimal values
-            string andahalf = " " + Properties.Phrases.andahalf;
-            switch (nextDigit)
+            if (morePreciseDecimalValues)
             {
-                case 0:
+                string andahalf = " " + Properties.Phrases.andahalf;
+                switch (nextDigit)
+                {
+                    case 0:
+                        return Properties.Phrases.justover + " " + minus + number + order;
+                    case 1:
+                        return Properties.Phrases.over + " " + minus + number + order;
+                    case 2:
+                        return Properties.Phrases.wellover + " " + minus + number + order;
+                    case 3:
+                        return Properties.Phrases.onthewayto + " " + minus + number + andahalf + order;
+                    case 4:
+                        return Properties.Phrases.nearly + " " + minus + number + andahalf + order;
```

Comment:
Some lacking of testing of the newly introduced cases, but more importantly I have to wonder if being this fine-grained on the next digit is justified.